ledger-core/virtual/integration/contract_calls_test.go

```diff
@@ -100,8 +101,9 @@ func Test_NoDeadLock_WhenOutgoingComeToSameNode(t *testing.T) {
 	for _, test := range table {
 		test := test
 		t.Run(test.name, func(t *testing.T) {
-			t.Log(test.testCaseID)
-			t.Skip("https://insolar.atlassian.net/browse/PLAT-432")
+			investigation.LogCase(t, test.testCaseID)
+			investigation.LogSkip(t, "https://insolar.atlassian.net/browse/PLAT-432")
```

Comment:
as these can't be separate, I'd suggest a form like 

investigation.Skip(t, "https://insolar.atlassian.net/browse/PLAT-432").Case(test.testCaseID)

Reply:
fixed